main.go

```diff
@@ -59,24 +60,45 @@ type autographer struct {
 }
 
 func main() {
+	run(parseArgsAndLoadConfig(os.Args[1:]))
+}
+
+func parseArgsAndLoadConfig(args []string) (conf configuration, listen string, authPrint, debug bool) {
 	var (
-		ag        *autographer
-		conf      configuration
-		cfgFile   string
-		debug     bool
-		authPrint bool
-		err       error
+		cfgFile string
+		port    int
+		err     error
+		fset    = flag.NewFlagSet("parseArgsAndLoadConfig", flag.ContinueOnError)
```

Comment:
This was to fix an error trying to add a duplicate flag when unit tests called parseArgsAndLoadConfig more than once.